Rakefile

```
@@ -38,7 +38,7 @@ task default: :test
 begin
   require 'rubocop/rake_task'
   RuboCop::RakeTask.new
-rescue => _
+rescue StandardError => _e
```

I'd be more specific in what you catch and not store the variable
```suggestion
rescue LoadError
```